app/src/Api/GameSerializer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Api;
+
+use Symfony\Component\Serializer\Serializer;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Normalizer\ArrayDenormalizer;
+use Symfony\Component\Serializer\Normalizer\GetSetMethodNormalizer;
+
+use App\Game\Game;
+
+class GameSerializer {
```

Comment:
You can implement the SerializerInterface to "fix" class implementation
http://api.symfony.com/3.1/Symfony/Component/Serializer/SerializerInterface.html